app/frontend/jest.config.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
   collectCoverageFrom: ['src/**/*.{js,vue}', '!src/main.js', '!src/plugins/*.*'],
   moduleFileExtensions: ['js', 'json', 'vue', 'jsx'],
   moduleNameMapper: {
-    '^@/(.*)$': '<rootDir>/src/$1'
+    '^@/(.*)$': '<rootDir>/src/$1',
+    '^axios$': require.resolve('axios') // import axios as module (ref: https://github.com/axios/axios/issues/5101)
```

Comment:
Do we need to be adding this in? We generally want to steer clear of editing the jest.config.js file unless it is a last resort. The frontend doesn't seem to be importing Jest at all in package.json - as if it was, you could update Jest to >29.2.x and it'd work out of the box. Ref: https://github.com/axios/axios/issues/5101#issuecomment-1291347644

It's been a while, but IIRC Jest is being imported as a sub-package managed by the vue/cli stuff. That can be updated to 4.15.19, which could potentially pull in a newer version of jest if that ecosystem was bumped potentially.